src/commands/synthetics/utils/public.ts

```diff
@@ -54,8 +54,8 @@ import {
 } from './internal'
 
 const POLLING_INTERVAL = 5000 // In ms
-const PUBLIC_ID_REGEX = /\b[a-z0-9]{3}-[a-z0-9]{3}-[a-z0-9]{3}\b/
 const TEMPLATE_REGEX = /{{\s*([^{}]*?)\s*}}/g
+export const PUBLIC_ID_REGEX = /\b[a-z0-9]{3}-[a-z0-9]{3}-[a-z0-9]{3}\b/
```

Comment:
Anything you export here will be exported publicly, and I would rather export a function to extract a publicId from a string, than exporting a regex constant.

Reply:
ah shit okay will probably pull it out then 👍